examples/android/routeguide/app/src/main/java/io/grpc/routeguideexample/RouteGuideActivity.java

```diff
@@ -166,7 +166,12 @@ protected String doInBackground(Void... nothing) {
                         RouteGuideGrpc.newStub(mChannel));
                 return "Success!" + System.lineSeparator() + logs;
             } catch (Exception e) {
-                return "Failed... : " + System.lineSeparator() + e.getMessage();
+                StringBuilder errorMessage = new StringBuilder("Failed: " + e.getMessage());
+                for (StackTraceElement element : e.getStackTrace()) {
```

Comment:
The one isn't enough, we need the full thing. Like we do in the android InteropTester.

```
StringWriter sw = new StringWriter();
PrintWriter pw = new PrintWriter(sw);
e.printStackTrace(pw);
pw.flush();
return "Failed... : " + System.lineSeparator() + sw;
```